apiary.apib

```diff
@@ -85,7 +85,7 @@ To ensure a fast and predictable experience for everyone, all our API endpoints
 
 When you go over this limit, our API will respond with a `HTTP 429 Too Many Requests error`.
 
-If your integrations needs a higher limit, send us an email at [api@teamleader.eu](mailto:api@teamleader.eu) with a detailed explanation why you need an increased rate limit.
+If your integrations needs a higher limit, send us an email at [api@teamleader.eu](mailto:api@teamleader.eu) with a detailed explanation of why you need an increased rate limit.
```

Comment:
`If your integrations neeD a higher limit`.

Reply:
"your integration needs" seems correct to me? 🤔

Reply:
That is correct, but it says `your integrationS` 🙈 	so then it needs to be `needS`

Reply:
oh yeah, I've read over the `s` in integrations. That makes it completely different, well spotted 🔍